### ISSUE TITLE
Seconds are not part of the usual cron spec! 

### ISSUE THREAD

**Issue by alonisser** [harrisiirak/cron-parser#38]
using them without explicit warning about that is quite broken.

I think that seconds should be allowed only with a specific flag/boolean, and not in causal parsing. just caused me an evil an unexpected bug

**Comment by harrisiirak** [comment]
Hi,

Sorry to hear that you had an issue with that. In the readme there is explicitly mentioned that seconds field is optional field. If you're not providing second field, it's working like specified in crontab standard, and if you're providing value for second field, it works in extended mode.

Can you provide me exact example which caused an issue?

BR,
Harri

**Comment by alonisser** [comment]
thanks for the kind reply @harrisiirak .I noticed that the docs specify seconds, unfortunately I noticed that only after I had the bug.. a quick visual look at the github page shows a drawing very similar to all  cron descriptions, but the rest don't have seconds.. that's why I missed that!. the basic problem was I was passing ``` 15 23 * * `1-5 *``` (23:15 monday-friday) so I had an extra ```*``` in the end of the cron expression. most cron implementations would shout at me or at least issue a warning.  (btw I'm not using directly but tried to debug node-schedule and the found the bug here)
